main.cpp

```diff
@@ -60,88 +61,158 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+    int age = 30;
     
+    int height = 6;
+
+    bool isOn = 0;
+
+    bool isGreen = 0;
+
+    bool isMetal = 0;
+
+    float bar = 0.f;
+
+    float phase = 0.f;
+
+    float volume = 0.f;
+
+    double dValue1 = 0.5;
+
+    double dValue2 = 1.0;
+
+    double dValue3 = 1.5;
+
+    char cValue1 = 0;
+
+    char cValue2 = 1;
+
+    char cValue3 = 2;
     
     
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
-}
+    ignoreUnused(number, age, height, isOn, isGreen, isMetal, bar, phase, volume, dValue1, dValue2, dValue3, cValue1, cValue2, cValue3); //passing each variable declared to the ignoreUnused() function
+} 
 /*
  10 functions
  example:
  */
 bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
 { 
     ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
-    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
-
+    return {}; //if your function returns something other than voi'return {};' at the end of it.} 
+}
 /*
  1)
  */
+bool playRecord(int recordName, int recordGenre = 0)
+{
+    ignoreUnused(recordName, recordGenre);
 
+    return{};
+
+}
 /*
  2)
  */
-
+void nameTrack(int artist, int genre, int tempo, int date)
+{
+    ignoreUnused(artist, genre, tempo, date);
+}
 /*
  3)
  */
-
+void run(int speed, int distance)
+{    
+    ignoreUnused(speed, distance);
+}
 /*
  4)
  */
-
+float createButton(int positionX, int positionY, int height, int length)
+{
+    ignoreUnused(positionX, positionY, height, length);
+    
+    return{};
+}
 /*
  5)
  */
-
+int displaySprite(int positionX, int positionY, int spriteType)
+{
+    ignoreUnused(positionX, positionY, spriteType);
+  
+    return {};
+}
 /*
  6)
  */
+float chooseVolume(int roomNumber, float averageVolume)
+{
+    ignoreUnused(roomNumber, averageVolume);
 
+    return{};
+}
 /*
  7)
  */
+float walkDog(bool yes, float distance)
+{
+    ignoreUnused(yes, distance);
 
+    return{};
+}
 /*
  8)
  */
+double createRemix(int track1, int track2, float finishedTempo)
+{
+    ignoreUnused(track1, track2, finishedTempo);
 
+    return{};
+}
 /*
  9)
  */
-
+void chooseJacket(int wheatherType, float exactTemprature)
+{
+    ignoreUnused(wheatherType, exactTemprature);
+}
 /*
  10)
  */
+int doPushups(int numPushups, int speed, bool withWeights)
+{
+    ignoreUnused(numPushups, speed, withWeights);
 
+    return{};
+}
```

Comment:
- [ ] do Instruction 4: add default values to an arbitrary number of function parameters.